src/pages/Unrelease.tsx

```diff
@@ -26,7 +27,8 @@ const validationSchema = Yup.object().shape({
         barcode: Yup.string().required(),
         highestSection: Yup.number().min(0, 'Section number must be greater than or equal to 0').notRequired()
       })
-    )
+    ),
+  workNumber: Yup.string().required('SGP Number is a required field')
```

Comment:
Is it mandatory? The story does not specify that.

Reply:
Ohh, is it? I thought it is mandatory. I will ask

Reply:
@khelwood  Katy has confirmed that the SGP number is mandatory